.eslintrc.js

```diff
@@ -113,7 +130,9 @@ module.exports = {
     },
     {
       files: ['website/**/*.{ts,tsx}'],
-      rules: { 'react/react-in-jsx-scope': 'off' },
+      rules: {
+        'react/react-in-jsx-scope': 'off',
+      },
     },
     {
       files: ['packages/**/*.spec.{ts,tsx}'],
```

Comment:
The props are not changing - the Storybook eslint plugin is just changing the formatting for some reason.